ci/jenkins/test.sh

```diff
@@ -670,6 +689,150 @@ function clean_tmp() {
     find ${WORKDIR} -name "support-bundles*" -mtime +7 -exec rm -rf {} \; 2>&1 | grep -v "Permission denied" || true
 }
 
+function deliver_k8s() {
```

Comment:
Why it is named "deliver_k8s"? set_k8s_ip_mode?

Reply:
This function will remove and redeploy K8s cluster, thus I follow the other functions' name (e.g. deliver_antrea)

Reply:
I feel we should name it redeploy_k8s().

Maybe add some comments about IP mode, etc.

Reply:
Renamed.
I updated parameter description for `ip-mode`. Do you think we still need more code comments?
`--ip-mode                IP mode for flexible ipam e2e test. Default is $DEFAULT_IP_MODE. It can also be ipv6 or ds."`

Reply:
I feel we should add comments about why/when we redeploy K8s.

Reply:
Added comment for this func.